crates/uv-python/fetch-download-metadata.py

```diff
@@ -106,6 +106,11 @@ class ImplementationName(StrEnum):
     PYPY = "pypy"
 
 
+class Variant(StrEnum):
+    FREETHREADED = "freethreaded"
+    DEBUG = "debug"
```

Comment:
In preparation for supporting download of debug builds too — I don't add that in the Rust side here though.